tracker/utils.py

```diff
@@ -215,3 +215,48 @@ def get_all_available_issues(url: str) -> list[dict]:
     except requests.exceptions.RequestException as e:
         logger.info(e)
     return []
+
+
+def get_pull_reviews(url: str) -> list[dict]:
```

Comment:
return type is not reponse to `return []`

Reply:
It still kind of applies in python typing. when it is not an empty list, it is a list of dicts. it matches both cases.